packages/react-storage/src/components/StorageBrowser/displayText/libraries/en/createFolderView.ts

```diff
@@ -6,11 +6,14 @@ export const DEFAULT_CREATE_FOLDER_VIEW_DISPLAY_TEXT: DefaultCreateFolderViewDis
     ...DEFAULT_ACTION_VIEW_DISPLAY_TEXT,
```

Comment:
Instead of always spreading shared text (which could grow and grow), I wonder if if should just be vended separately and the consumers still just pick and choose. Something like:
```ts
const { MyView, ActionViewCommon, Common } = useDisplayText()
const { textSpecificToMyView } = MyView;
const { textSharedBySomeActionViews } = ActionViewCommon;
const { textRandomlySharedAllOver } = Common;
```
I'm not sure the benefits of this is really worth it but just a suggestion to consider

Reply:
Yeah, not seeing a lot of utility in the shared objects as i've gone farther with the display text but is something we can address later